ports/osgearth/portfile.cmake

```
@@ -51,10 +53,15 @@ vcpkg_cmake_configure(
         -DOSGEARTH_BUILD_ZIP_PLUGIN=OFF		
         -DWITH_EXTERNAL_TINYXML=ON
         -DCMAKE_JOB_POOL_LINK=console # Serialize linking to avoid OOM
+        -DOSGEARTH_BUILD_LEGACY_CONTROLS_API=${WITH_CONTROLS}
     OPTIONS_DEBUG
         -DOSGEARTH_BUILD_TOOLS=OFF
     MAYBE_UNUSED_VARIABLES
         LIB_POSTFIX
+        WITH_BLEND2D
+        WITH_EXAMPLES
+        WITH_EXTERNAL_TINYXML
+        WITH_CONTROLS
```

```suggestion
        WITH_BLEND2D
        WITH_EXTERNAL_TINYXML
```
but it should be investigated what it means that the remaining variables are unused.